pkg/network/protocols/kafka/telemetry.go

```diff
@@ -27,37 +23,19 @@ func NewTelemetry() *Telemetry {
 	metricGroup := libtelemetry.NewMetricGroup(
 		"usm.kafka",
 		libtelemetry.OptExpvar,
-		libtelemetry.OptMonotonic,
 	)
 
-	t := &Telemetry{
-		then: atomic.NewInt64(time.Now().Unix()),
-
+	return &Telemetry{
```

Comment:
you didn't set metricGroup here, it will cause a crash in `log`

Reply:
Good catch! 🙇